examples/default/mongodb/mongodb_test.go

```diff
@@ -25,7 +25,7 @@ import (
 	eventstore "github.com/looplab/eventhorizon/eventstore/mongodb"
 	readrepository "github.com/looplab/eventhorizon/readrepository/mongodb"
 
-	"github.com/looplab/eventhorizon/examples/domain"
+	"github.com/looplab/eventhorizon/examples/default/domain"
```

Comment:
If possible it would be nice to share the domain logic even with custom ID types. Maybe we can set the ID type to use in each "main" file (example_test.go)? That would require switching the NewID to a factory (and maybe have CreateID for creating blank IDs) similar to the events and aggregate factories recently added.